index.html

```diff
@@ -117,6 +145,30 @@ <h3 class="crete-font">Multi-Post Stories</h3>
         </a>
       </div>
     </div>
+
+    <!-- <div class="modal" id="myModal">
+      <div class="modal-content"></div>
+      <h2 class="crete-font">Multi-Post Stories</h2>
+      <span class="close">&times;</span>
+      <ul class="list inter-font">
+        <li>html</li>
+        <li>bootstrap</li>
+        <li>Ruby on rails</li>
+      </ul>
+      <img class="pop-img" src="imgs/Snapshoot-Portfolio.png" alt="">
+      <p class="inter-font">
+        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent
+      </p>
+      <div class="pop-buttons" id="myBtn">
+      <a href="#" class="inter-font btns">
+        See Live
+      </a>
```

Comment:
* kindly remove commented code from your codebase is make your codebase bulky and unprofessional

Reply:
This particular session has now been removed.
Thanks